lambda-resume-builder/src/Components/Register/Register.js

```diff
@@ -85,9 +85,17 @@ const Register = (props) =>  {
     setState({...state, [event.target.name]:event.target.value});
   }
 
+  const redirect = () => {
+    console.log('Props', props)
+    window.location.pathname = '/dashboard';
```

Comment:
### Direct manipulation of the pathname is a start, but I think we can find another way eventually.